src/kubectl.ts

```diff
@@ -194,6 +194,10 @@ async function spawnAsChild(context: Context, command: string[]): Promise<ChildP
 
 async function invokeInTerminal(context: Context, command: string, pipeTo: string | undefined, terminal: Terminal): Promise<void> {
     if (await checkPresent(context, 'command')) {
+        // You might be tempted to think we needed to add 'wsl' here if user is using wsl
+        // but this runs in the context of a vanilla terminal, which is controlled by the
+        // existing preference, so it's not necessary.
+        // But a user does need to default VS code to use WSL in the settings.json
```

Comment:
Might be good to check for this and provide a warning notification if they are using a PowerShell terminal with a WSLised extension, as this could cause weird errors e.g. a resource appears in the extension, but commands that interact with it in the terminal fail because the Windows kubeconfig has a different active cluster from the WSL kubeconfig.

Reply:
Can we get at vs code preferences like this?